python/cudf/cudf/core/column/string.py

```diff
@@ -1769,6 +1768,17 @@ def __init__(self, mask=None, size=None, offset=0, children=()):
             None, size, dtype, mask=mask, offset=offset, children=children
         )
 
+        # For an "all empty" StringColumn (e.g., [""]) libcudf still
+        # needs the chars child column pointer to be non-null:
+        if self.size:
+            if self.null_count == 0 and self.children[1].size == 0:
```

Comment:
Don't we need to do this if `self.null_count != self.size`? I.e. `cudf.Series(["", None])`

Reply:
Fixed